addon/styles/_pix-app-layout.scss

```
@@ -42,7 +42,7 @@
 
   &__main, &__footer {
     width: 100%;
-    max-width: 93rem;
+    max-width: calc(100vw - 2 * var(--pix-spacing-6x));
```

```suggestion
    max-width:  min(calc(100vw - 2 * var(--pix-spacing-6x)), 93rem);
```

A priori ça fait ce que l'on souhaite.